Microsoft.Toolkit.Uwp.UI.Controls/RotatorTile/RotatorTile.cs

```diff
@@ -37,7 +37,6 @@ public sealed class RotatorTile : Control
         private const string TranslatePartName = "Translate";
         private const string StackPartName = "Stack";
 
-        private static readonly Random _randomizer = new Random(); // randomizer for randomizing when a tile swaps content
```

Comment:
Randomization was an integral part so several tiles don't appear in sync. This is also a breaking change in behavior. This should be kept but I'm OK with introducing another Duration property for randomization amount